.gitlab-ci.yml

```diff
@@ -144,6 +145,19 @@ build_android:
       - jniLibs
     expire_in: 3 mos
 
+build_python:
+  stage: build
+  image:
+    name: ghcr.io/pyo3/maturin
+    # override the image entrypoint
```

Comment:
It's better to explain why you need to override than explaining what you do (we can see thanks to the following line what you do)